docs/modules/ROOT/pages/includes/quarkus-langchain4j-tavily.adoc

```diff
@@ -124,194 +124,7 @@ ifndef::add-copy-button-to-env-var[]
 Environment variable: `+++QUARKUS_LANGCHAIN4J_TAVILY_SEARCH_DEPTH+++`
 endif::add-copy-button-to-env-var[]
 --
-a|SearchDepth
-|`basic`
-
-a| [[quarkus-langchain4j-tavily_quarkus-langchain4j-tavily-include-answer]] [.property-path]##link:#quarkus-langchain4j-tavily_quarkus-langchain4j-tavily-include-answer[`quarkus.langchain4j.tavily.include-answer`]##
-
-[.description]
---
-Include a short answer to original query. Default is false.
-
-
```

Comment:
Talking about this one and the others below.

Reply:
This certainly shouldn't be happenning, huh.
I've also noticed in https://github.com/quarkiverse/quarkus-langchain4j/tree/main/docs/modules/ROOT/pages/includes that all the files seem to be duplicated for some reason

Reply:
Yes this is expected. You have one global file per extension and then one per root. Most of the time, they are the same but we have extensions with multiple roots.

Reply:
Hmmm, actually, that's normal because in the current doc, the properties are duplicated. They must have been generated when there were two modules in a local tree with this root or something similar (or a YAML model from a previous run in another directory).

Look for `include-answer` here: https://github.com/quarkiverse/quarkus-langchain4j/blob/main/docs/modules/ROOT/pages/includes/quarkus-langchain4j-tavily.adoc .

So I think you can merge this one, it's all good.

Reply:
Cool, thanks for checking